src/io.rs

```diff
@@ -0,0 +1,137 @@
+use std::collections::{HashMap, HashSet};
+
+use abstio::MapName;
+use abstutil::Timer;
+use anyhow::{bail, Result};
+use convert_osm::reader;
+use enum_map::{enum_map, EnumMap};
+use geom::Distance;
+use raw_map::{DrivingSide, RawIntersection, RawMap, RawRoad};
+use serde::Deserialize;
+
+use crate::network::RoadNetwork;
+use crate::road_functions::IntersectionType;
+use crate::road_functions::{ControlType, Intersection, RoadWay};
+use crate::road_parts::RoadEdge;
+use crate::units::preamble::{Backward, Forward, Left, Right};
+use crate::units::Direction;
+
+// use crate::osm_geom::{get_multipolygon_members, glue_multipolygon, multipoly_geometry};
+
+/// ```
+/// use abstutil::Timer;
+/// use petgraph::dot::{Config, Dot};
+/// use streets::io::load_road_network;
+/// let mut timer = Timer::new("test osm2streets");
+/// let mut net = load_road_network(String::from("tests/src/aurora_sausage_link/input.osm"), &mut timer);
+/// println!("{}", Dot::new(&map.graph));
+pub fn load_road_network(path: String, timer: &mut Timer) -> RoadNetwork {
+    let driving_side = DrivingSide::Left; // TODO get driving side from country containing lat lon?
+    let clip = None;
+
+    let mut raw_map = convert_osm::convert(
+        path,
+        MapName::new("zz", "osm2streets_test", &abstutil::basename(&osm_path)),
+        clip,
+        convert_osm::Options {
```

Comment:
I'm considering a `convert_osm::Options::common_settings(driving_side)` here; most of these are default/boilerplate. (Not sure about implementing `Default`, since `driving_side` really is required)